addNewCharacters.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+require('functions.php');
+$db = connect_db('CartoonCollection');
+$newTvShowName = extract_tvShow_from_db($db);
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Add New Character to Collection</title>
+    <link href="cartoons-stylesheet.css" type="text/css" rel="stylesheet">
+    <link href="normalize.css" type="text/css" rel="stylesheet">
+    <link href="cartoon-font.ttf" type="text/ttf" rel="stylesheet">
+    <link href="BalooTammudu2-Regular.ttf" type="text/ttf" rel="stylesheet">
+</head>
+
+<body>
+
+<div class="goBack">
+    <a href="index.php" rel="prev">< Go Back</a>
+</div>
+
+<div class="newCharacterTitle">
+    <h1>Add a New Character</h1>
+</div>
+
+<div class="formNewCharacter">
+    <form method="POST" action="addNewItems_toDb.php">
+        <label for="characterName"><strong>Character Name:</strong></label>
+        <br>
```

Comment:
woah, whats with all the `<br>` tags? what happened to using `<div>`s with margin?